logging/telemetryhook.go

```diff
@@ -165,6 +169,10 @@ func createElasticHook(cfg TelemetryConfig) (hook logrus.Hook, err error) {
 	}
 	hostName := cfg.getHostName()
 	hook, err = elogrus.NewElasticHook(client, hostName, cfg.MinLogLevel, cfg.ChainID)
+
+	if err == nil && hook == nil {
```

Comment:
That's pretty untypical for go; usually the object and the error are mutually exclusive.

Reply:
I think that this doesn't actually happen and I was confused initially because nil, nil is returned when initialized with an empty telemetry string. I'll remove this and make sure that is really the case.